functional-test/src/main/java/org/zanata/page/projects/projectsettings/ProjectLanguagesTab.java

```diff
@@ -43,8 +39,10 @@
 @Slf4j
 public class ProjectLanguagesTab extends ProjectBasePage {
 
-    private By addNewLanguageField = By.id("languageAutocomplete-autocomplete__input");
-    private By settingsLanguagesForm = By.id("settings-languages-form");
+    private By activeLocales = By.id("active-locales-list");
+    private By inactiveLocales = By.id("available-locales-list");
```

Comment:
At this point I realize that the `id` of the components doesn't help. Part of cleaning up that code will be to make all those consistently use `enabled` and `disabled` as well.